relayer/message_coordinator.go

```diff
@@ -252,3 +240,20 @@ func (mc *MessageCoordinator) ProcessBlock(blockHeader *types.Header, ethClient
 		go appRelayer.ProcessHeight(block.BlockNumber, handlers, errChan)
 	}
 }
+
+func FetchWarpMessageFromID(ethClient ethclient.Client, warpID common.Hash, blockNum *big.Int) (*relayerTypes.WarpMessageInfo, error) {
```

Comment:
Naming nit: `FetchWarpMessage` seems sufficient to describe the function. Having the Warp ID in both the function name and as a parameter seems redundant.